FalmouthFalcons/iteration-4
# Made some change to all buttons and styling as well as homepage makeover

Description:
Made major styling changes to buttons and layout. Also changed homepage to look like a store

Number of Fixes:
Several pages changed

Related Ticket(s):
N/A

Problem to Solve:
Look of site and buttons

Proposed Changes:
N/A

Expected Behavior:
All buttons and links should work

Steps to Test Solution
1) git pull origin master
2) git fetch
3) git checkout jim_#6_delete_from_cart

Documentation
[ ] There is new documentation in this pull request that must be reviewed..
[ ] I added documentation for any new classes/methods

Deploy Notes:
- Notes regarding deployment the contained body of work.
- These should note any db migrations, etc.
